src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -39,7 +39,10 @@ export default View.extend({
       links = links.filter(l => $.isPlainObject(l));
     }
 
-    if (this.options.appState.get('showSignoutLink')) {
+    if (this.options.settings.get('features.hideSignOutLinkInMFA') &&
+        this.options.appState.get('isVerifyIdentityForm')) {
+      // do not add cancel/signout link if the config is on and it is a verify identity form
+    } else if (this.options.appState.get('showSignoutLink')) {
```

Comment:
why can't we just check for !features.hideSignOutLinkInMFA here?

Reply:
because the config is for MFA only, it is not for all flows. So I don't want to completely hide it from all forms if the feature is enabled, I only want to hide it for the MFA forms.

Reply:
If i understand this correct, we wont send `features.hideSignOutLinkInMFA` from the server every-time? We will only send it as true for MFA forms ie 
```
FORMS.SELECT_AUTHENTICATOR_AUTHENTICATE
FORMS.CHALLENGE_AUTHENTICATOR
```

Reply:
okta hosted widget will always send that config, but it will be `false` in all flows EXCEPT enrolling an authenticator from end user settings page.

Reply:
2 questions 

- If its going to be true for enrolling an authenticator from end user settings page should these be ? 

```
FORMS.SELECT_AUTHENTICATOR_ENROLL,
FORMS.ENROLL_AUTHENTICATOR
```

- Secondly in that case can we just check if  `features.hideSignOutLinkInMFA` is true ? As the backend knows to set this  to true for only these flow

Reply:
no, `ENROLL_AUTHENTICATOR` is already on the list of `FORMS_WITHOUT_SIGNOUT`, and that is an enroll form.
the config `hideSignOutLinkInMFA` only affects challenge forms, that is why I need to add `CHALLENGE_AUTHENTICATOR`

Not sure I understand your second point. We were discussing this in chat, since this config is only meant for challenge/verify forms, I cannot just check for that alone, I need to also be sure it is a form that is affected by that config

Reply:
another opinion is change `showSignoutLink ` to be a function instead of derive props ?
```
showSignoutLink(hideSignOutLinkInMFA) {
 ...
}
```

Reply:
yeah, that way we can keep all the checks in one place

Reply:
done